datasafe-rest/src/main/java/de/adorsys/datasafe/rest/config/DatasafeProperties.java

```diff
@@ -0,0 +1,13 @@
+package de.adorsys.datasafe.rest.config;
+
+import lombok.Data;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+
+@ConfigurationProperties(prefix = "datasafe")
+@Data
+public class DatasafeProperties {
+
+    String bucketName;
```

Comment:
Fields should be private

Reply:
ok. I thought @data is making them private